sympy/calculus/util.py

```diff
@@ -373,6 +375,16 @@ def periodicity(f, symbol, check=False):
 
         period = _periodicity(g.args, symbol)
 
+    elif isinstance(f, Mod):
+        a, n = f.args
+
+        if a == symbol:
+            period  = n
```

Comment:
The extra space to the left of the `==` can be moved to up above (in the test file) where there are no spaces ;-) (lines 88-90 and line 104.

Reply:
Oops. Edited accordingly. :)

Reply:
Oops. Edited accordingly. :)